diff --git a/tools/releasetools/ota_from_target_files.py b/tools/releasetools/ota_from_target_files.py
index f52b212..283be32 100755
--- a/tools/releasetools/ota_from_target_files.py
+++ b/tools/releasetools/ota_from_target_files.py
@@ -742,26 +742,26 @@ else if get_stage("%(bcb_dev)s") == "3/3" then
     manufacturer = GetBuildProp("ro.product.manufacturer", OPTIONS.info_dict)
     maintainer = GetBuildProp("ro.build.user", OPTIONS.info_dict)
     sdkver = GetBuildProp("ro.build.version.sdk", OPTIONS.info_dict)
-    script.Print(" **************** Software *****************");
-    script.Print(" OS ver: %s"%(buildid));
-    script.Print("");
-    script.Print(" Android ver: %s"%(androidver));
-    script.Print("");
-    script.Print(" Security patch: %s"%(securep));
-    script.Print("");
-    script.Print(" SDK ver: %s"%(sdkver));
-    script.Print("");
-    script.Print(" Root status: Enabled");
-    script.Print("");
-    script.Print(" Build ID: %s"%(buildidn));
-    script.Print("");
-    script.Print(" Build date: %s"%(buildday));
-    script.Print("");
-    script.Print(" Build type: %s"%(buildtype));
-    script.Print("");
-    script.Print(" Build host: %s"%(buildhst));
-    script.Print("");
-    script.Print(" Maintainer: %s"%(maintainer));
+    #script.Print(" **************** Software *****************");
+    #script.Print(" OS ver: %s"%(buildid));
+    #script.Print("");
+    #script.Print(" Android ver: %s"%(androidver));
+    #script.Print("");
+    #script.Print(" Security patch: %s"%(securep));
+    #script.Print("");
+    #script.Print(" SDK ver: %s"%(sdkver));
+    #script.Print("");
+    #script.Print(" Root status: Enabled");
+    #script.Print("");
+    #script.Print(" Build ID: %s"%(buildidn));
+    #script.Print("");
+    #script.Print(" Build date: %s"%(buildday));
+    #script.Print("");
+    #script.Print(" Build type: %s"%(buildtype));
+    #script.Print("");
+    #script.Print(" Build host: %s"%(buildhst));
+    #script.Print("");
+    #script.Print(" Maintainer: %s"%(maintainer));
     script.Print(" **************** Hardware *****************");
     script.Print(" Device codename: %s"%(device));
     script.Print("");
